### PR TITLE
fix(test): add screenshots for login

### PR DESCRIPTION
**Pull request by ldimaggi**

### CODE REVIEW

File: ee_tests/src/page_objects/login.page.ts [hunk 1]
```diff
@@ -47,10 +47,14 @@ export class LoginPage extends BasePage {
     await this.passwordInput.enterText(password);
     await this.loginButton.clickWhenReady();
 
+    support.writeScreenshot('target/screenshots/login_1.png');
```
Comment (ppitonak) [inline ee_tests/src/page_objects/login.page.ts]: It doesn't make sense to take a screenshot immediately after clicking the button... the page will be always in transition state at this point. Move it before `await this.loginButton.clickWhenReady()`

File: ee_tests/src/page_objects/login.page.ts [hunk 2]
```diff
@@ -47,10 +47,14 @@ export class LoginPage extends BasePage {
...
+
     this.debug('... Login: input details and click Login - OK');
 
     let mainDashboard = new MainDashboardPage()
     await mainDashboard.open()
```
Comment (ppitonak) [inline ee_tests/src/page_objects/login.page.ts]: Although not related to this PR, please add semicolons to these two lines

File: ee_tests/src/page_objects/login.page.ts (hunk 1 above)
Comment (ldimaggi) [reply ee_tests/src/page_objects/login.page.ts]: Done

File: ee_tests/src/page_objects/login.page.ts (hunk 2 above)
Comment (ldimaggi) [reply ee_tests/src/page_objects/login.page.ts]: Done

**Pull request merged**